tests/integration/schema_registry/data/invalid.json

```diff
@@ -0,0 +1,31 @@
+{
```

Comment:
Can we indicate somehow where the invalid part is, because it is not appearant when reading this?